### ISSUE TITLE
Release 4.2 M5 (2021.0.0)

### ISSUE THREAD

**Issue by sothawo** [spring-projects/spring-data-elasticsearch#1709]

**Comment by morganlutz** [comment]
Hi @sothawo 👋 . Do you know when this will be released? Or - is the `dense_vector` change available in a snapshot version we can use? Thank you 🙏

**Comment by sothawo** [comment]
According to https://github.com/spring-projects/spring-data-commons/wiki/Release-Train-2021.0-(Pascal) M5 is planned for March 17th.

But the change is already available in the 4.2.0-SNAPSHOT that is built each night. Should be
```xml
        <repository>
            <id>spring-libs-snapshot</id>
            <name>Spring Snapshot Repository</name>
            <url>https://repo.spring.io/snapshot</url>
        </repository>
```

**Comment by morganlutz** [comment]
thank you!
